index.html

```diff
@@ -5,39 +5,49 @@
   <title>All Random All Root</title>
   <meta name="description" content="All Random All Root">
   <meta name="author" content="Ryan Haswell">
-  <style>
-  ul.seat-list {list-style-type: decimal}
-  </style>
+  <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Roboto">
+
+  <link rel="stylesheet" href="styles/main.css"/>
 </head>
 
 <body>
-  <h1>ALL RANDOM ALL ROOT</h1>
+  <div class="main">
+    <h1>ALL RANDOM ALL ROOT</h1>
 
-  <h4>Enter your parameters below, or die</h4>
+    <div class="entry-form">
+      <div class="entry-form-body">
+        <h4>Enter your parameters below, or die</h4>
 
-  <div>
-    <h5>Players:</h5>
-    <ul id="playerList">
-    </ul>
-  </div>
+        <form onSubmit="addPlayer(event)">
+          <label for="playerNameInput">Player Name:</label>
+          <input id="playerNameInput" name="name">
+          <input type="submit" value="Add Player" class="button">
+        </form>
+
+        <div class="submissions">
+          <form onSubmit="clearPlayers(event)">
+            <input type="submit" value="Clear Players" class="button primary">
+          </form>
 
-  <form onSubmit="addPlayer(event)">
-    <label for="playerNameInput">Player Name:</label>
-    <input id="playerNameInput" name="name">
-    <input type="submit" value="Add Player">
-  </form>
+          <form onSubmit="generateGame(event)">
+            <input type="submit" value="Randomize" class="button secondary">
+          </form>
+        </div>
+      </div>
+    </div>
 
-  <form onSubmit="clearPlayers(event)">
-    <input type="submit" value="Clear Players">
-  </form>
 
-  <form onSubmit="generateGame(event)">
-    <input type="submit" value="Randomize">
-  </form>
+    <div id="output"></div>
 
-  <div id="output"></div>
 
-  <script src="js/data.js"></script>
-  <script src="js/app.js"></script>
+    <div>
```

Comment:
Can you move the player list back above the output? Otherwise goes off screen. I think player list should be visible next to the clear players/randomize buttons